### ISSUE TITLE
[BUG] IndexError: list index out of range when running lonboard CLI on gpkg file

### ISSUE THREAD

**Issue by robintw** [developmentseed/lonboard#433]
## Context

I'm running the lonboard CLI on a GeoPackage file as follows:

```
lonboard West_Devon_rail.gpkg
```

## Resulting behaviour, error message or logs

I get the following output:

```
Traceback (most recent call last):
  File "/Users/robin/mambaforge/envs/biosouth/bin/lonboard", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/lonboard/_cli.py", line 121, in main
    table = read_pyogrio(path)
            ^^^^^^^^^^^^^^^^^^
  File "/Users/robin/mambaforge/envs/biosouth/lib/python3.12/site-packages/lonboard/_cli.py", line 33, in read_pyogrio
    geometry_column_index = [
                            ^
IndexError: list index out of range
```

## Environment

- OS: MacOS 12.7.3
- Browser: Chrome (but it crashes before it gets to the browser)
- Lonboard Version: 0.7.0

## Steps to reproduce the bug

1. Download `West_Devon_rail.gpkg` from https://automaticknowledge.org/gb/rail/West_Devon_rail.gpkg
2. Run `lonboard West_Devon_rail.gpkg`

**Comment by kylebarron** [comment]
Thanks for the report! This should be fixed in 0.7.1

**Comment by robintw** [comment]
Wonderful, thanks for the very prompt fix!
